constants/monitor.js

```diff
@@ -0,0 +1,15 @@
+const RESOURCE_CREATED_SUCCEEDED = "Resource created successfully.";
+const RESOURCE_UPDATED_SUCCEEDED = "Resource updated successfully.";
+const RESOURCE_RETRIEVAL_SUCCEEDED = "Resource retrieved successfully.";
```

Comment:
why cannot we put the constant

Reply:
I didn't get you. Can you reword your doubt again.

Reply:
```suggestion
const RESOURCE : {
CREATED_SUCCEEDED: "Resource created successfully.";
UPDATED_SUCCEEDED : "Resource updated successfully.";
RETRIEVAL_SUCCEEDED : "Resource retrieved successfully.";
}
```

Reply:
@bhtibrewal @Pratiyushkumar if you see below I am already exporting as `RESPONSE_MESSAGES.RESOURCE_CREATED_SUCCEEDED` , `RESPONSE_MESSAGES. RESOURCE_UPDATED_SUCCEEDED`. Let me know if you still feel it needs to be changed.

Reply:
Randhir don't you think we need to put variable names in present tense instead oof past tense for eg. :-  instead of  RESOURCE_CREATED_SUCCEEDED, can't we write RESOURCE_CREATED_SUCCESSFULLY

Reply:
Yes randhir I saw that export at the end, but I still think we don't need to write RESOURCE with every constant here so making a nested object like I suggested above, removes that repeating word.

Reply:
@bhtibrewal then in that case it will be `RESPONSE_MESSAGES.RESOURCE.CREATED_SUCCEEDED`. I think just for a simple message we are adding unnecessary 2 levels of nesting.